src/groups.py

```diff
@@ -34,8 +34,12 @@ def expand_nodes(node_list):
     del nodes[-1]
     return nodes
 
-def __nodeattr(file_path='{}genders'.format(config.LEADER), arguments=[], split_char="\n"):
+def __nodeattr(file_path='{}genders'.format(config.LEADER),
```

Comment:
Can we save the genders file path in the `config`?  Something like `config.GENDERS_FILE`

Reply:
Good shout: done here https://github.com/alces-software/adminware/pull/169/commits/453d2bc59b7ab8b96cf397d0ba5545857788ea5d